bin/bashrc

```diff
@@ -22,8 +25,13 @@ export TYCHO2_KD_DIR=/global/project/projectdirs/cosmo/staging/tycho2
 export PS1CAT_DIR=/global/project/projectdirs/cosmo/work/ps1/cats/chunks-qz-star-v3/
 
 # desiconda
-module use /global/common/software/desi/$NERSC_HOST
-module load desiconda/20180512-1.2.5-img
+if [ $NERSC_HOST == "edison" ]; then
+  module use /global/common/software/desi/edison/desiconda/20180512-1.2.5-img/modulefiles
+  module load desiconda/20180512-1.2.5-img
+else # cori
+  module use /global/cscratch1/sd/desiproc/DRcode/desiconda/software/modulefiles
+  module load desiconda/20180511
+fi
 
```

Comment:
We should be using the same conda environment on both machines.  
```
desiconda_version=20180512-1.2.5-img
module use /global/common/software/desi/$NERSC_HOST/desiconda/$desiconda_version/modulefiles
module load desiconda
```
should work fine.